prow/config/generate.go

```diff
@@ -581,7 +581,7 @@ func applyRequirements(job *config.JobBase, requirements []string) {
 			)
 			job.Spec.Containers[0].VolumeMounts = append(job.Spec.Containers[0].VolumeMounts,
 				v1.VolumeMount{
-					MountPath: "/go/pkg",
+					MountPath: "/home/prow/go/pkg",
 					Name:      "build-cache-pvc",
 					SubPath:   "gomod",
```

Comment:
I missed this before, but should this be `mod` instead of `gomod`?

Reply:
this part doesn't matter, its just the path on the actual NFS drive. as long as its consistent its fine, the mountPath is what matters